backpack/extensions/firstorder/batch_grad/__init__.py

```diff
@@ -89,3 +89,6 @@ def get_subsampling(self) -> Union[List[int], None]:
             means all samples will be considered.
         """
         return self._subsampling
+
+    def expects_backpropagation_quantities(self) -> bool:  # noqa: D102
+        return False
```

Comment:
Can this be moved to a `FirstOrderBackpropExtension` base class that inherits from `BackpropExtension`? Same for the other first-order extensions

Reply:
Created a base class.